bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -462,6 +462,20 @@ public function getAction(Request $request, EventDispatcherInterface $eventDispa
 
             $validLayouts = DataObject\Service::getValidLayouts($object);
 
+            //Fallback if $currentLayoutId is not set or empty string
+            //Uses first valid layout instead of admin layout when empty
+            $ok = false;
+            foreach ($validLayouts as $key => $layout){
+                if($currentLayoutId == $layout->getId()){
+                    $ok = true;
+                }
+            }
+            if(!$ok){
+                if (count($validLayouts) > 0) {
+                    $currentLayoutId = $validLayouts[array_key_first($validLayouts)]->getId();
```

Comment:
maybe use https://www.php.net/manual/en/function.reset.php instead ?

Reply:
Looks better now, thanks. Didn't know this function at all.